hail/scripts/benchmark_in_batch.py

```diff
@@ -110,7 +110,7 @@ def make_benchmark_trial(
     j.command('mkdir -p benchmark-resources')
     j.command(
         ' '.join([
-            f'python3 -m pytest {path}',
+            f'python3 -m pytest \'{path}::{benchmark_name}\'',
```

Comment:
Tiny nit, but the entire point of python allowing both single and double quoted strings is so that you don't need to escape the quotes.
```suggestion
            f"python3 -m pytest '{path}::{benchmark_name}'",
```